src/bindings/python/CMakeLists.txt

```diff
@@ -170,7 +170,7 @@ if(UNIX)
 	set_property(TARGET PyOpenColorIO PROPERTY POSITION_INDEPENDENT_CODE ON)
 endif()
 
-if (UNIX AND NOT CMAKE_SKIP_RPATH AND NOT CMAKE_INSTALL_RPATH)
+if (UNIX AND NOT CMAKE_SKIP_RPATH AND CMAKE_INSTALL_RPATH)
```

Comment:
Suggest to rollback to the condition we had in [RB-2.2](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/RB-2.2/src/bindings/python/CMakeLists.txt#L173) and do the same for [CompilerFlags](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/RB-2.2/share/cmake/utils/CompilerFlags.cmake#L100).
`if (UNIX AND NOT CMAKE_SKIP_RPATH)`

I originally was trying to have the [Python build](https://github.com/AcademySoftwareFoundation/OpenColorIO/blob/main/setup.py#L158) only ship with the required RPATH and nothing more but it turned out it created this bug and I also later realised that the wheel have a "repair" step at the end, which for macOS simply replace the RPATH entirely (as can be seen in GHA logs; I assume Linux is similar but the log is quieter) so these were probably not really needed.

Reply:
@KevinJW What do you think about simplifying the above as suggested in the earlier comment? Would be great to have this in the upcoming release @doug-walker is planning for this week.